webpack.config.js

```diff
@@ -60,10 +60,6 @@ module.exports = {
       enforce: 'post'
     }],
   },
-  externals: {
```

Comment:
this change seems unrelated, just pointing it out in case you didn't mean to add it here